### PR TITLE
News article about case 2020-0007

### PR DESCRIPTION
**Pull request by MrSeccubus**
@Lennaert89 please review.

### CODE REVIEW

File: _posts/20-05-28-Citrix-ShareFile.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+---
+<small>[English below](#english)</small>
+
+### Nederlands
+
+Op 5 mei heeft Citrix klanten die gebruik maken van de Citrix ShareFile oplossing gewaarschuwd over drie kwetsbaarheden in dit product en hiervoor patches en een mitigatie [beschikbaar](https://support.citrix.com/article/CTX269341) gesteld. 
+
+Citrix heeft geen technisch inhoudelijke mededelingen gedaan over de verholpen kwetsbaarheden, maar onderzoek van [Dimitri van de Giessen](https://twitter.com/DimitriNL) wijst erop dat een van de kwetsbaarheden is ontstaan omdat Citrix gebruik gemaak heeft van een oude Ajax component met een kwetsbaarheid die het een aanvaller mogelijk maakt vertrouwelijke bestanden te lezen.
+
+Opvallend is dat de het uitvoeren van de patch er slechts voor zorgt dat de kwetsbaarheid niet meer voorkomt in nieuwe "storage zones" die aangemaakt worden na het patchen. Om de kwetsbaarheid ook te verwijderen in eerder gecreerde "storage zones" is het nodige mitigerende acties uit te voeren op alle storage servers in het cluster.
```
Comment (Lennaert89) [inline _posts/20-05-28-Citrix-ShareFile.md]: gecreëerde

File: _posts/20-05-28-Citrix-ShareFile.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Ernstige lek in Citrix ShareFIle storage server / Critical vulnerability in Citrix ShareFIle storage server
+author: Frank Breedijk
+excerpt:  Citrix ShareFIle servers bevatten een ernstig lek waarmee een aanvaller gevoelige informatie kan achterhalen / Citrix ShareFile servers contain a vulnerability that allows an attacker to obtain sensitive information.
```
Comment (Lennaert89) [inline _posts/20-05-28-Citrix-ShareFile.md]: ShareFile

File: _posts/20-05-28-Citrix-ShareFile.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+### English
+
+On May 5th Citrix has informed customers that use the Citrix ShareFile solution of three vulnerabilities in this product and has made patches and mitigation tools [available](https://support.citrix.com/article/CTX269341) to adress them. 
+
+Citrix has not disclosed technical details about these vulnerabilities, but reseacher [Dimitri van de Giessen](https://twitter.com/DimitriNL) has published that at least one of the vulnerabilities is the result of the use of an outdated Ajax component with a vulnerability that allow an attacker to access confidential files.
+
+Please take note that applying the patches only resolves this vulnerabilty for "storage zones" that are newly created after the patch has been applied. To resolve the vulnerability for pre-existing "storage zones", a mitigation tool must be applied to all storage servers in the cluster.
+
+[DIVD](https://www.divd.nl) researchers have scanned for vulnerable, Dutch, ShareFIle server. [The Security Hotline](https://www.securitymeldpunt.nl) will contact the administrators of these servers.
```
Comment (Lennaert89) [inline _posts/20-05-28-Citrix-ShareFile.md]: ShareFile

**Review by Lennaert89 — CHANGES_REQUESTED** [review]
Few spelling mistakes/errors/typos, looks good for the rest.

File: _posts/20-05-28-Citrix-ShareFile.md (hunk 3 above)
Comment (MrSeccubus) [reply _posts/20-05-28-Citrix-ShareFile.md]: Fixed

**Pull request merged**